recipes/asap/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set ASAP_VERSION = "1.6" %}
+package:
+    name: asap
+    version: {{ ASAP_VERSION }}
+
+source:
+    fn: {{ ASAP_VERSION }}.zip
+    url: https://github.com/GeertLitjens/ASAP/archive/{{ ASAP_VERSION }}.zip
```

Comment:
Yes, for the exact version your are downloading we need a checksum to verify that the download was successful and no harm was done with the tarball.